test/MUXControlsReleaseTest/MUXControls.ReleaseTest/NugetTestsCX.cs

```diff
@@ -29,6 +29,12 @@ public static void ClassInitialize(TestContext testContext)
             TestEnvironment.Initialize(testContext, TestType.NugetCX);
         }
 
+        [TestCleanup]
+        public void TestCleanup()
+        {
+            PGOManager.PGOSweepIfInstrumented(TestEnvironment.TestContext.TestName);
```

Comment:
Do we need to sweep for every test method, or can we do it all at once in AssemblyCleanup?

Reply:
I like to keep this separate and the count files smaller. 

---
In reply to: [301736142](https://github.com/microsoft/microsoft-ui-xaml/pull/988#discussion_r301736142) [](ancestors = 301736142)